README.md

```diff
@@ -1 +1,272 @@
-# Taskcluster configuration for Servo
+# Taskcluster configuration for Servo CI
+
+This repository contains tools and configuration for Servo’s use of
+[Taskcluster](https://community-tc.services.mozilla.com/docs/) for testing pull requests.
+
+As much as possible of the configuration is kept “in tree”
+[in the `servo/servo` repository](https://github.com/servo/servo/tree/master/etc/taskcluster)
+so that changes (for example `apt install`’ing a new system dependency in a `Dockerfile`)
+can be made in the same pull request that relies on them.
+This reduces the need for coordination across multiple repositories.
+See also the `README` there for more on Servo’s testing workflow.
+
+Some other parts of the configuration are not practical to deploy automatically
+when merging a pull request, and require semi-manual steps. They are kept here.
+
+
+## Requirements
+
+Ensure that you have `python3` and `virtulenv` executables in your `$PATH`.
```

Comment:
Typo: "virtualenv". Do these need to match? (That is, your virtualenv has to be the one for that python3?)